frontend/src/pages/ApprovalConfirmation.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { Button, Form, Container } from 'react-bootstrap';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+
+const Wrapper = styled(Container)`
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+`;
+
+const Title = styled.h4`
+  margin: 10px;
+`;
+
+const StepActionsWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row-reverse;
+`;
+
+const ConfirmationCheckbox = (props: { onChange: (checked: boolean) => void }) => {
+  const auth = useKeycloakWrapper();
+
+  const label = `I, ${auth.firstName} ${auth.lastName}, confirm by checking this box that I have received the approval 
+    to request the property be submitted to the Enhanced Referral Program and marketed internally to other Ministry 
+    and Broader Public Sector Agencies for 90 days before listing the property to be sold on the public commercial real estate market`;
+  return (
+    <Form.Group controlId="formBasicCheckbox">
+      <Form.Check
+        type="checkbox"
+        label={label}
+        onChange={(event: any) => props.onChange(event.target.value)}
+      />
+    </Form.Group>
+  );
+};
+
+const StepActions = (props: { onSave: () => void; onNext: () => void }) => {
+  return (
+    <StepActionsWrapper>
+      <Button style={{ marginLeft: 10 }} onClick={props.onNext}>
+        Next
+      </Button>
+      <Button onClick={props.onSave}>Save</Button>
+    </StepActionsWrapper>
+  );
+};
+
+export const ApprovalConfirmation = () => {
+  // const api = useApi();
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [confirm, setConfirm] = React.useState(false);
+
+  const save = () => {};
+
+  const next = () => {};
+
+  return (
+    <Wrapper>
+      <Title>Approval Confirmation</Title>
+      <ConfirmationCheckbox onChange={checked => setConfirm(checked)} />
+      <StepActions onNext={next} onSave={save} />
```

Comment:
@goemen I think it makes the most sense if you just write this component independently of the others I'm working on, and then I handle integrating it with my code. As a result, I'm fine with this as-is

Reply:
On the integration part, is there any input props or events that this component has to support?

Reply:
Like for instance, I have to check if the project was approved by exemption or not, and display different text on the page. How do I make this check?